recipes/rnalien/build.sh

```diff
@@ -7,5 +7,8 @@ export CPPFLAGS="-I${PREFIX}/include"
 stack setup
 stack update
 stack install --extra-include-dirs ${PREFIX}/include --local-bin-path ${PREFIX}/bin
+mv ${PREFIX}/bin/RNAlien ${PREFIX}/bin/RNAlien-bin
```

Comment:
You could just create such a bash script:
```
#!/bin/bash
export EMBOSS_ACDROOT=../share/EMBOSS/acd/
export EMBOSS_DATA=../share/EMBOSS/data
export PLPLOT_LIB=../share/EMBOSS
_water $@
```

and copy it from RECIPE_DIR into the $PREFIX.